src/app.ts

```diff
@@ -29,6 +30,7 @@ const nunjucksEnv = nunjucks.configure([
   "views/update/remove",
   "node_modules/govuk-frontend/",
   "node_modules/govuk-frontend/components",
+  "node_modules/@companieshouse/"
```

Comment:
Just wondering if / on the end is required? Seems to be there in some lines above and not others. Maybe we can save a byte!